setup.py

```diff
@@ -65,6 +65,8 @@
 os_files = [
     # XDG application description
     ('share/applications', ['xdg/openshot-qt.desktop']),
+    # AppStream metadata
+    ('share/appdata', ['xdg/openshot-qt.appdata.xml']),
```

Comment:
Apparently [these should be stored](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#spec-component-location) in `/usr/share/metainfo/` now; `/usr/share/appdata/` is also read for backwards compatibility but it's deprecated.